### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by rprzedzik**

### CODE REVIEW

File: module/workflow/src/Infrastructure/Query/Decorator/WorkflowGetProductQueryDecorator.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            $result['status'] = $status;
+            foreach ($transitions as $transition) {
+                $result['workflow'] = [];
+                if ($this->service->available($transition, $product)) {
+                    $result['workflow'][] = [
+                        'name' => $transition->getName()->get($language),
+                        'status' => $transition->getDestination(),
+                        '_links' => [
+                            'update' => [
+                                'href' => 'change product status url if needed',
```
Comment (BastekBielawski) [inline module/workflow/src/Infrastructure/Query/Decorator/WorkflowGetProductQueryDecorator.php]: Why?

File: module/workflow/src/Infrastructure/Grid/TransitionGrid.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        $column = new TextColumn('name', $this->trans('Name'), new TextFilter($filters->getString('name')));
+        $column->setWidth(200);
+        $this->addColumn('name', $column);
+
+        $column = new TextColumn('description', $this->trans('Description'), new TextFilter($filters->getString('description')));
+        $column->setWidth(300);
+        $this->addColumn('description', $column);
+
+        $url1 = sprintf('/api/v1/%s/workflow/default/transitions/{source}/{destination}', $language->getCode());
```
[the remark this answers is not in the document]
Comment (rprzedzik) [reply module/workflow/src/Infrastructure/Grid/TransitionGrid.php]: This is template for object that at this moment don have access to services

File: features/workflow_transitions.feature [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    Given current authentication token
+    When I request "/api/v1/EN/workflow/default/transitions/@workflow_source_status_code@/@workflow_destination_status_code@" using HTTP DELETE
+    Then empty response is received
+
+  Scenario: Get transition in default Workflow
+    Given current authentication token
+    When I request "/api/v1/EN/workflow/default/transitions/@workflow_source_status_code@/@workflow_destination_status_code@"
+    Then the response code is 404
+
+  Scenario: Delete default status
```
Comment (BastekBielawski) [inline features/workflow_transitions.feature]: Check "not found" and "not authorized"

File: module/condition/src/Domain/Provider/ConditionCalculatorProvider.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (BastekBielawski) [inline module/condition/src/Domain/Provider/ConditionCalculatorProvider.php]: Add comment and declaration
```
/**
 * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE.txt for license details.
 */

declare(strict_types = 1);
```

File: module/condition/src/Domain/Service/Strategy/Calculator/NumericAttributeValueConditionCalculatorStrategy.php [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<?php
```
Comment (BastekBielawski) [inline module/condition/src/Domain/Service/Strategy/Calculator/NumericAttributeValueConditionCalculatorStrategy.php]:
```
/**
 * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE.txt for license details.
 */

declare(strict_types = 1);
```

File: module/condition/src/Domain/Service/Strategy/Calculator/OptionAttributeValueConditionCalculatorStrategy.php [hunk 6]
```diff
@@ -0,0 +1,63 @@
+<?php
```
Comment (BastekBielawski) [inline module/condition/src/Domain/Service/Strategy/Calculator/OptionAttributeValueConditionCalculatorStrategy.php]:
```
/**
 * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE.txt for license details.
 */

declare(strict_types = 1);
```

File: module/condition/src/Domain/Service/Strategy/Calculator/TextAttributeValueConditionCalculatorStrategy.php [hunk 7]
```diff
@@ -0,0 +1,66 @@
+<?php
```
Comment (BastekBielawski) [inline module/condition/src/Domain/Service/Strategy/Calculator/TextAttributeValueConditionCalculatorStrategy.php]:
```
/**
 * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE.txt for license details.
 */

declare(strict_types = 1);
```

**Pull request merged**